resources/views/index.php

```diff
@@ -50,12 +50,25 @@
                 }
             });
 
+            var rowsById = {};
+            $('.table tbody tr').each(function() {
+                rowsById[$(this).attr('id')] = $(this);
+            });
+            
+            $('#search-field').keyup(function() {
+                var searchKey = $(this).val().toUpperCase();
+                Object.keys(rowsById).forEach(function(key) {
+                    rowsById[key].toggle(!searchKey || key.includes(searchKey));
```

Comment:
You need to replace key.includes(searchKey) with key.toUpperCase().includes(searchKey). 
Otherwise very good. Good Job.